doc/contribute/project_roles.rst

```diff
@@ -90,6 +90,22 @@ in addition to those listed for Contributors and Collaborators:
 * Responsibility to enforce the code of conduct.
 
 
+Role Retirement
+###############
+
+* Contributors elected to leadership roles (Maintainer,
+  Release Engineer Team member, Release Manager, etc.)
+  are expected to be engaged in the project as described
+  by their rights and responsibilities.
+* Contributors elected to a leadership role, but are no
+  longer engaged in the project as described by the rights
+  and responsibilities of that role, may be requested by the
+  TSC to retire from the leadership role. Such a request needs
```

Comment:
Make it clear that the process is a TSC member would raise a motion for removal and that would be voted.

Reply:
Agreed